### PR TITLE
Update Profile README

### PR DESCRIPTION
**Pull request by chirag-ghosh**
- [x] add about
- [x] add cover image
- [x] add social links

### CODE REVIEW

**Comment by hargup** [comment]
LGTM

**Review by hargup — DISMISSED** [review]
LGTM

File: profile/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  
+## What does it do? 
+
+- Maintains the student-led community wiki of IIT Kharagpur.
+- Creates and maintains projects which makes the life of a KGPian easier.
+- Provides an open forum for anyone to join and discuss anything related to technology.
+
+
+## Contact
+- Mail - `metakgp@gmail.com`
```
Comment (proffapt) [inline profile/README.md]: isko mailto: nahi banana?

File: profile/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# Metakgp
+
+![metakgp-wall](./wall.png)
+
+## What is it?
+
+An open community coming together to help solve problems an average KGPian faces in everyday life using tech. Issues can be anything that is related to KGP in general. It is an amalgamation of the [Wiki](https://wiki.metakgp.org) and multiple projects the students have been building over the years.
```
Comment (proffapt) [inline profile/README.md]:
```suggestion
An open community coming together to help solve problems an average KGPian faces in everyday life using tech. Issues can be anything that is related to KGP in general. It is an amalgamation of the [Wiki](https://wiki.metakgp.org) and multiple projects which the students have been building over the years.
```

**Review by proffapt — CHANGES_REQUESTED** [review]
Shouldn't it be like..
- Who are we?
- What do we do?

**Comment by chirag-ghosh** [comment]
> Shouldn't it be like..
> 
> * Who are we?
> * What do we do?

Wanted to define the community as a whole. Can change the phrasing if people think this is more appropriate.

**Comment by proffapt** [comment]
> > Shouldn't it be like..
> > 
> > * Who are we?
> > * What do we do?
> 
> Wanted to define the community as a whole. Can change the phrasing if people think this is more appropriate.

Got your essence but I feel this is much better way of achieving what you are saying. Using `it` kinda does the opposite 🤔

**Review by thealphadollar — DISMISSED** [review]

**Review by proffapt — APPROVED** [review]
LGTM.

**Review by rajivharlalka — APPROVED** [review]

**Pull request merged**